base/cartesian.jl

```diff
@@ -98,6 +98,19 @@ function _nall(N::Int, criterion::Expr)
     Expr(:&&, conds...)
 end
 
+# Check whether any of variables i1, i2, ... satisfy criterion
+macro nany(N, criterion)
+    _nany(N, criterion)
+end
+
+function _nany(N::Int, criterion::Expr)
```

Comment:
I'd add a couple, but I can't find tests for any of the existing macros in ``Cartesian``.